src/utilities/IOManager.cpp

```diff
@@ -97,19 +97,24 @@ const proto::SSL_Referee &IOManager::getRefereeData() {
 
 void IOManager::publishRobotCommand(proto::RobotCommand cmd) {
     if (!pause->getPause()) {
-        // the geneva cannot be received from world, so we set it when it gets sent.
-        auto robot = world_new::World::instance()->getWorld()->getRobotForId(cmd.id(), true);
-        if (robot) {
-            if (cmd.kicker()) {
-                interface::Input::drawData(interface::Visual::SHOTLINES, {robot->get()->getPos()}, Qt::green, robot->get()->getId(), interface::Drawing::CIRCLES, 36, 36, 8);
+        if (world_new::World::instance()->getWorld()) {
+            // the geneva cannot be received from world, so we set it when it gets sent.
+            auto robot = world_new::World::instance()->getWorld()->getRobotForId(cmd.id(), true);
+            if (robot) {
+                if (cmd.kicker()) {
+                    interface::Input::drawData(interface::Visual::SHOTLINES, {robot->get()->getPos()}, Qt::green,
+                            robot->get()->getId(), interface::Drawing::CIRCLES, 36, 36, 8);
```

Comment:
Make sure you use constants for 36 and 8. These are totally undocumented magic values, which will cause huge problems in the future...